geomstats/geometry/quotient_metric.py

```diff
@@ -61,17 +62,19 @@ def inner_product(
         inner_product : float, shape=[...]
             Inner products
         """
-        if point is None:
+        if point_fiber is None:
```

Comment:
if statements can be merged?

Reply:
The else condition would need to be replaced by `elif fiber_point is None and base_point is None` so it's just as clear?